<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoAdjustClass</class>
 <widget class="QWidget" name="AutoAdjustClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>914</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1413</width>
    <height>914</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1413</width>
    <height>914</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>自动调校系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string>串口设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>串口号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="box_serial_id"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="box_baud">
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbtn_ascll">
           <property name="text">
            <string>ASCLL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn_hex">
           <property name="text">
            <string>HEX</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="btn_open_serial">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>实时视频</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_picture">
        <property name="minimumSize">
         <size>
          <width>960</width>
          <height>720</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>960</width>
          <height>720</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>实时数据区</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>特征1：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fea1">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>29</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>特征2：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fea2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>特征3：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fea3">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>29</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>特征4：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fea4">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>特征点</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>04b_21</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>选取机芯:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbox_feature">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_fea_id">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_learn_feature">
        <property name="minimumSize">
         <size>
          <width>255</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>学习特征</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>自动调校系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>相机</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_open_camera">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>打开相机</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_catch_pic">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>捕获图像</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_ctrl_status">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_stop">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>日志</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QTextBrowser" name="text_log"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_catch_log">
        <property name="text">
         <string>保存日志</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_clear_log">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutoAdjust.qrc"/>
 </resources>
 <connections/>
</ui>
